[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-k8s/0 [idle] active: 
  mysql-k8s/1 [idle] active: Primary
  mysql-k8s/2 [idle] active: 
[32mINFO    [0m juju.model:model.py:3211 Waiting for model:
  mysql-test-app/0 [executing] active: 
  mysql-test-app/1 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:790 Model status:

Model    Controller          Cloud/Region        Version  SLA          Timestamp
testing  concierge-microk8s  microk8s/localhost  3.6.6    unsupported  01:31:23Z

App             Version                  Status   Scale  Charm           Channel      Rev  Address         Exposed  Message
mysql-k8s       8.0.41-0ubuntu0.22.04.1  active       3  mysql-k8s                      0  10.152.183.41   no       
mysql-test-app  0.0.2                    waiting      2  mysql-test-app  latest/edge   73  10.152.183.183  no       

Unit               Workload  Agent  Address    Ports  Message
mysql-k8s/0        active    idle   10.1.8.77         
mysql-k8s/1*       active    idle   10.1.8.78         Primary
mysql-k8s/2        active    idle   10.1.8.79         
mysql-test-app/0   waiting   idle   10.1.8.72         
mysql-test-app/1*  waiting   idle   10.1.8.73         

[32mINFO    [0m pytest_operator.plugin:plugin.py:796 Juju error logs:

unit-mysql-k8s-1: 01:29:48 ERROR unit.mysql-k8s/1.juju-log get_unit_address: unit DNS domain name is not fully propagated yet, trying again
unit-mysql-k8s-1: 01:29:50 ERROR unit.mysql-k8s/1.juju-log get_unit_address: unit DNS domain name is not fully propagated yet, trying again
unit-mysql-k8s-1: 01:30:26 ERROR unit.mysql-k8s/1.juju-log database:4: get_unit_address: unit DNS domain name is not fully propagated yet, trying again
unit-mysql-k8s-1: 01:30:28 ERROR unit.mysql-k8s/1.juju-log database:4: get_unit_address: unit DNS domain name is not fully propagated yet, trying again
unit-mysql-k8s-1: 01:30:30 ERROR unit.mysql-k8s/1.juju-log database:4: get_unit_address: unit DNS domain name is not fully propagated yet, trying again

[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Forgetting main...